general/package/ingenic-osdrv-t31/files/sensor/gc4653.yaml

```diff
@@ -4,3 +4,4 @@ sensor:
   width: 2560
   height: 1440
   bus: i2c
+  resolutions: 2560x1440x25_400,1920x1080x60_300,1280x720x60_100
```

Comment:
If we want to keep it .yaml then syntax should be
```
resolution:
  - 2560x1440x25_400
  - 1920x1080x60_300
  - 1280x720x60_100
```
Otherwise there is no point to keep it .yaml, a simple .ini syntax would work better.

Reply:
> If we want to keep it .yaml then syntax should be
> 
> ```
> resolution:
>   - 2560x1440x25_400
>   - 1920x1080x60_300
>   - 1280x720x60_100
> ```
> 
> Otherwise there is no point to keep it .yaml, a simple .ini syntax would work better.

I tried not to produce new configs, and immediately wrote it into the sensor config.

Reply:
I didn't mean it a new config. Just the notation. An array of values in yaml is one per line with a dash at the beginning of the line item.